go/vt/vtgate/engine/helpers.go

```diff
@@ -0,0 +1,61 @@
+package engine
```

Comment:
license missing

Reply:
Moved contents to `go/vt/vtgate/planbuilder/operators/helpers.go` for now, which already has a license.